src/oc_erchef/apps/oc_chef_wm/itest/oc_chef_wm_cookbook_artifact_SUITE.erl

```diff
@@ -125,11 +125,8 @@ db_round_trip(Config) ->
     ?assert(erlang:is_integer(RecIDFromDB)),
 
     %% and modulo a few things, it should be equal to what we inserted
-    ChecksumsFromDB = RecFromDB#oc_chef_cookbook_artifact_version.checksums,
```

Comment:
Interesting. Why is this related to the postgres version change?

Reply:
There was  a verify pipeline error.  Short story: A checksum list came back in reverse order, and therefore did not match because of this, causing an error.  So I corrected the issue.  Longer story:  The checksum list actually came back in correct order, but previously (sometime in the past) PG sent the list back in reverse order, so we apparently had a line that accounted for that.  PG apparently fixed the issue, so the list no longer comes back reversed, so I had to remove our patch which accounted for that.

expected:
```
                    [<<"36a1cc1b225708db96d48026c3f624b2">>,
                     <<"45b27c78955f6a738d2d42d88056c57c">>,
                     <<"967087a09f48f234028d3aa27a094882">>,
                     <<"ba0dadcbca26710a521e0e3160cc5e20">>,
                     <<"c92b659171552e896074caa58dada0c2">>,
                     <<"d18c630c8a68ffa4852d13214d0525a6">>]
```
received:
```
                    [<<"d18c630c8a68ffa4852d13214d0525a6">>,
                     <<"c92b659171552e896074caa58dada0c2">>,
                     <<"ba0dadcbca26710a521e0e3160cc5e20">>,
                     <<"967087a09f48f234028d3aa27a094882">>,
                     <<"45b27c78955f6a738d2d42d88056c57c">>,
                     <<"36a1cc1b225708db96d48026c3f624b2">>]
```
full error:
```
    correct verify pipeline error

    the checksum list at the end was reversed, causing the error.

    %%% oc_chef_wm_cookbook_artifact_SUITE ==> db_round_trip: FAILED
    %%% oc_chef_wm_cookbook_artifact_SUITE ==>
    Failure/Error: ?assertEqual({oc_chef_cookbook_artifact_version,0,undefined,<<100,98,95,114,111,117,110,100,95,116,114,105,112,95,105,100,101,110,116,105,1
02,105,101,114>>,<<31,139,8,0,0,0,0,0,0,3,85,144,91,107,2,49,16,133,255,74,200,243,146,90,31,3,133,46,86,208,82,20,122,121,170,69,98,118,140,129,205,76,200,69
,40,226,127,239,168,171,109,31,2,147,228,156,111,102,206,65,118,16,1,59,64,235,33,75,125,144,169,110,190,165,254,252,106,206,149,131,144,79,183,99,35,209,4,14
4,90,86,244,150,18,202,70,6,227,177,240,129,180,6,174,123,254,164,152,31,237,14,182,202,19,11,76,41,201,111,106,57,131,153,208,123,11,152,79,144,54,26,150,137
,177,26,177,44,87,231,32,23,79,56,232,98,111,202,150,82,24,174,191,125,216,185,140,217,82,7,141,152,163,101,111,79,232,214,29,100,155,124,60,17,88,241,32,94,2
30,147,233,226,109,42,218,197,147,104,63,222,103,203,87,189,194,21,182,181,236,40,105,45,218,206,4,241,108,44,109,148,82,12,73,96,41,4,78,193,252,25,98,15,41,
95,128,35,117,175,198,44,179,132,91,94,161,120,116,23,9,251,124,188,164,118,77,69,203,57,230,98,250,62,139,225,73,12,49,74,54,184,68,53,178,61,95,253,188,169,
189,225,254,175,113,229,220,77,184,173,119,53,193,13,201,179,196,68,123,223,13,214,227,15,134,29,153,149,198,1,0,0>>,<<31,139,8,0,0,0,0,0,0,3,141,147,65,111,2
20,32,16,133,255,11,231,213,46,140,1,227,220,42,53,199,94,122,173,42,107,24,160,161,217,53,22,198,145,162,40,255,189,108,227,221,172,179,217,170,7,75,134,25,1
27,239,61,15,188,48,231,67,28,98,137,105,152,216,221,143,23,54,224,193,179,59,54,15,145,82,30,122,74,67,136,191,182,217,178,13,163,7,79,143,211,124,168,101,23
4,192,106,213,137,86,40,5,222,116,154,183,146,16,149,113,232,144,19,212,238,17,203,67,237,188,224,239,62,131,78,163,167,24,34,197,242,252,214,141,243,190,176,
215,159,27,134,165,228,104,231,226,143,198,234,58,196,253,233,117,204,233,41,58,159,151,229,62,218,140,57,158,170,197,31,198,61,190,125,247,49,80,21,221,250,1
43,105,26,141,130,72,88,0,213,114,227,108,167,157,52,28,52,53,65,131,180,23,105,206,232,221,226,116,119,197,189,29,40,251,41,205,153,78,54,23,99,206,246,57,20
5,131,235,107,218,177,255,187,185,97,53,220,80,42,197,231,171,142,139,210,17,73,113,92,7,93,20,175,70,102,145,215,225,144,37,4,221,10,142,10,132,231,190,17,15
4,19,41,15,252,61,228,2,221,173,72,255,136,149,82,233,79,195,57,219,248,126,255,229,235,183,251,109,118,137,214,62,156,48,164,27,78,6,181,9,1,165,81,224,68,3,
66,58,174,64,161,126,247,177,70,220,48,176,57,43,30,124,169,135,175,224,85,242,78,215,169,182,200,187,32,77,128,70,114,48,174,65,132,227,150,52,230,98,188,107
,196,255,43,254,158,210,176,214,148,202,66,75,173,233,148,10,26,219,198,56,112,18,156,49,92,105,82,45,125,162,185,64,110,255,232,138,15,125,121,30,143,210,148
,210,163,173,79,143,185,30,6,164,210,63,213,235,80,111,25,123,253,3,248,131,168,29,212,3,0,0>>,{datetime,{{2021,5,24},{5,51,33}}},<<48,48,48,48,48,48,48,48,48
,48,48,48,48,48,48,48,48,48,48,48,48,48,48,48,48,48,48,48,48,48,48,53>>,<<101,57,102,51,52,102,48,100,50,99,100,50,99,99,50,99,53,102,101,56,102,100,56,51,50,
100,49,98,56,101,51,55>>,<<100,98,95,114,111,117,110,100,95,116,114,105,112,95,110,97,109,101>>,<<56,54,55,49,50,52,55,102,100,54,49,54,48,50,56,54,51,49,52,9
7,48,57,55,56,57,56,53,57,100,98,52,98>>,[<<51,54,97,49,99,99,49,98,50,50,53,55,48,56,100,98,57,54,100,52,56,48,50,54,99,51,102,54,50,52,98,50>>,<<52,53,98,50
,55,99,55,56,57,53,53,102,54,97,55,51,56,100,50,100,52,50,100,56,56,48,53,54,99,53,55,99>>,<<57,54,55,48,56,55,97,48,57,102,52,56,102,50,51,52,48,50,56,100,51
,97,97,50,55,97,48,57,52,56,56,50>>,<<98,97,48,100,97,100,99,98,99,97,50,54,55,49,48,97,53,50,49,101,48,101,51,49,54,48,99,99,53,101,50,48>>,<<99,57,50,98,54,
53,57,49,55,49,53,53,50,101,56,57,54,48,55,52,99,97,97,53,56,100,97,100,97,48,99,50>>,<<100,49,56,99,54,51,48,99,56,97,54,56,102,102,97,52,56,53,50,100,49,51,
50,49,52,100,48,53,50,53,97,54>>]}, ModifiedRecFromDB)
      expected: {oc_chef_cookbook_artifact_version,0,undefined,
                    <<"db_round_trip_identifier">>,
                    <<31,139,8,0,0,0,0,0,0,3,85,144,91,107,2,49,16,133,255,74,200,
                      243,146,90,31,3,133,46,86,208,82,20,122,121,170,69,98,118,
                      140,129,205,76,200,69,40,226,127,239,168,171,109,31,2,147,
                      228,156,111,102,206,65,118,16,1,59,64,235,33,75,125,144,169,
                      110,190,165,254,252,106,206,149,131,144,79,183,99,35,209,4,
                      144,90,86,244,150,18,202,70,6,227,177,240,129,180,6,174,123,
                      254,164,152,31,237,14,182,202,19,11,76,41,201,111,106,57,131,
                      153,208,123,11,152,79,144,54,26,150,137,177,26,177,44,87,231,
                      32,23,79,56,232,98,111,202,150,82,24,174,191,125,216,185,140,
                      217,82,7,141,152,163,101,111,79,232,214,29,100,155,124,60,17,
                      88,241,32,94,230,147,233,226,109,42,218,197,147,104,63,222,
                      103,203,87,189,194,21,182,181,236,40,105,45,218,206,4,241,
                      108,44,109,148,82,12,73,96,41,4,78,193,252,25,98,15,41,95,
                      128,35,117,175,198,44,179,132,91,94,161,120,116,23,9,251,124,
                      188,164,118,77,69,203,57,230,98,250,62,139,225,73,12,49,74,
                      54,184,68,53,178,61,95,253,188,169,189,225,254,175,113,229,
                      220,77,184,173,119,53,193,13,201,179,196,68,123,223,13,214,
                      227,15,134,29,153,149,198,1,0,0>>,
                    <<31,139,8,0,0,0,0,0,0,3,141,147,65,111,220,32,16,133,255,11,
                      231,213,46,140,1,227,220,42,53,199,94,122,173,42,107,24,160,
                      161,217,53,22,198,145,162,40,255,189,108,227,221,172,179,217,
                      170,7,75,134,25,127,239,61,15,188,48,231,67,28,98,137,105,
                      152,216,221,143,23,54,224,193,179,59,54,15,145,82,30,122,74,
                      67,136,191,182,217,178,13,163,7,79,143,211,124,168,101,234,
                      192,106,213,137,86,40,5,222,116,154,183,146,16,149,113,232,
                      144,19,212,238,17,203,67,237,188,224,239,62,131,78,163,167,
                      24,34,197,242,252,214,141,243,190,176,215,159,27,134,165,228,
                      104,231,226,143,198,234,58,196,253,233,117,204,233,41,58,159,
                      151,229,62,218,140,57,158,170,197,31,198,61,190,125,247,49,
                      80,21,221,250,143,105,26,141,130,72,88,0,213,114,227,108,167,
                      157,52,28,52,53,65,131,180,23,105,206,232,221,226,116,119,
                      197,189,29,40,251,41,205,153,78,54,23,99,206,246,57,205,131,
                      235,107,218,177,255,187,185,97,53,220,80,42,197,231,171,142,
                      139,210,17,73,113,92,7,93,20,175,70,102,145,215,225,144,37,4,
                      221,10,142,10,132,231,190,17,154,19,41,15,252,61,228,2,221,
                      173,72,255,136,149,82,233,79,195,57,219,248,126,255,229,235,
                      183,251,109,118,137,214,62,156,48,164,27,78,6,181,9,1,165,81,
                      224,68,3,66,58,174,64,161,126,247,177,70,220,48,176,57,43,30,
                      124,169,135,175,224,85,242,78,215,169,182,200,187,32,77,128,
                      70,114,48,174,65,132,227,150,52,230,98,188,107,196,255,43,
                      254,158,210,176,214,148,202,66,75,173,233,148,10,26,219,198,
                      56,112,18,156,49,92,105,82,45,125,162,185,64,110,255,232,138,
                      15,125,121,30,143,210,148,210,163,173,79,143,185,30,6,164,
                      210,63,213,235,80,111,25,123,253,3,248,131,168,29,212,3,0,0>>,
                    {datetime,{{2021,5,24},{5,51,33}}},
                    <<"00000000000000000000000000000005">>,
                    <<"e9f34f0d2cd2cc2c5fe8fd832d1b8e37">>,
                    <<"db_round_trip_name">>,
                    <<"8671247fd6160286314a09789859db4b">>,
                    [<<"36a1cc1b225708db96d48026c3f624b2">>,
                     <<"45b27c78955f6a738d2d42d88056c57c">>,
                     <<"967087a09f48f234028d3aa27a094882">>,
                     <<"ba0dadcbca26710a521e0e3160cc5e20">>,
                     <<"c92b659171552e896074caa58dada0c2">>,
                     <<"d18c630c8a68ffa4852d13214d0525a6">>]}
           got: {oc_chef_cookbook_artifact_version,0,undefined,
                    <<"db_round_trip_identifier">>,
                    <<31,139,8,0,0,0,0,0,0,3,85,144,91,107,2,49,16,133,255,74,200,
                      243,146,90,31,3,133,46,86,208,82,20,122,121,170,69,98,118,
                      140,129,205,76,200,69,40,226,127,239,168,171,109,31,2,147,
                      228,156,111,102,206,65,118,16,1,59,64,235,33,75,125,144,169,
                      110,190,165,254,252,106,206,149,131,144,79,183,99,35,209,4,
                      144,90,86,244,150,18,202,70,6,227,177,240,129,180,6,174,123,
                      254,164,152,31,237,14,182,202,19,11,76,41,201,111,106,57,131,
                      153,208,123,11,152,79,144,54,26,150,137,177,26,177,44,87,231,
                      32,23,79,56,232,98,111,202,150,82,24,174,191,125,216,185,140,
                      217,82,7,141,152,163,101,111,79,232,214,29,100,155,124,60,17,
                      88,241,32,94,230,147,233,226,109,42,218,197,147,104,63,222,
                      103,203,87,189,194,21,182,181,236,40,105,45,218,206,4,241,
                      108,44,109,148,82,12,73,96,41,4,78,193,252,25,98,15,41,95,
                      128,35,117,175,198,44,179,132,91,94,161,120,116,23,9,251,124,
                      188,164,118,77,69,203,57,230,98,250,62,139,225,73,12,49,74,
                      54,184,68,53,178,61,95,253,188,169,189,225,254,175,113,229,
                      220,77,184,173,119,53,193,13,201,179,196,68,123,223,13,214,
                      227,15,134,29,153,149,198,1,0,0>>,
                    <<31,139,8,0,0,0,0,0,0,3,141,147,65,111,220,32,16,133,255,11,
                      231,213,46,140,1,227,220,42,53,199,94,122,173,42,107,24,160,
                      161,217,53,22,198,145,162,40,255,189,108,227,221,172,179,217,
                      170,7,75,134,25,127,239,61,15,188,48,231,67,28,98,137,105,
                      152,216,221,143,23,54,224,193,179,59,54,15,145,82,30,122,74,
                      67,136,191,182,217,178,13,163,7,79,143,211,124,168,101,234,
                      192,106,213,137,86,40,5,222,116,154,183,146,16,149,113,232,
                      144,19,212,238,17,203,67,237,188,224,239,62,131,78,163,167,
                      24,34,197,242,252,214,141,243,190,176,215,159,27,134,165,228,
                      104,231,226,143,198,234,58,196,253,233,117,204,233,41,58,159,
                      151,229,62,218,140,57,158,170,197,31,198,61,190,125,247,49,
                      80,21,221,250,143,105,26,141,130,72,88,0,213,114,227,108,167,
                      157,52,28,52,53,65,131,180,23,105,206,232,221,226,116,119,
                      197,189,29,40,251,41,205,153,78,54,23,99,206,246,57,205,131,
                      235,107,218,177,255,187,185,97,53,220,80,42,197,231,171,142,
                      139,210,17,73,113,92,7,93,20,175,70,102,145,215,225,144,37,4,
                      221,10,142,10,132,231,190,17,154,19,41,15,252,61,228,2,221,
                      173,72,255,136,149,82,233,79,195,57,219,248,126,255,229,235,
                      183,251,109,118,137,214,62,156,48,164,27,78,6,181,9,1,165,81,
                      224,68,3,66,58,174,64,161,126,247,177,70,220,48,176,57,43,30,
                      124,169,135,175,224,85,242,78,215,169,182,200,187,32,77,128,
                      70,114,48,174,65,132,227,150,52,230,98,188,107,196,255,43,
                      254,158,210,176,214,148,202,66,75,173,233,148,10,26,219,198,
                      56,112,18,156,49,92,105,82,45,125,162,185,64,110,255,232,138,
                      15,125,121,30,143,210,148,210,163,173,79,143,185,30,6,164,
                      210,63,213,235,80,111,25,123,253,3,248,131,168,29,212,3,0,0>>,
                    {datetime,{{2021,5,24},{5,51,33}}},
                    <<"00000000000000000000000000000005">>,
                    <<"e9f34f0d2cd2cc2c5fe8fd832d1b8e37">>,
                    <<"db_round_trip_name">>,
                    <<"8671247fd6160286314a09789859db4b">>,
                    [<<"d18c630c8a68ffa4852d13214d0525a6">>,
                     <<"c92b659171552e896074caa58dada0c2">>,
                     <<"ba0dadcbca26710a521e0e3160cc5e20">>,
                     <<"967087a09f48f234028d3aa27a094882">>,
                     <<"45b27c78955f6a738d2d42d88056c57c">>,
                     <<"36a1cc1b225708db96d48026c3f624b2">>]}
          line: 138
    ...........
    %%% oc_chef_wm_keys_SUITE: ...............................................
    %%% oc_chef_wm_policies_SUITE: .............
    %%% oc_chef_wm_sanboxes_SUITE: ....
    %%% oc_chef_wm_server_api_version_SUITE: ............
    EXPERIMENTAL: Writing retry specification at /workdir/src/oc_erchef/_build/test/logs/retry.spec
                  call rebar3 ct with '--retry' to re-run failing cases.
    Failed 1 tests. Passed 127 tests.
    Results written to "/workdir/src/oc_erchef/_build/test/logs/index.html".
    ===> Failures occurred running tests: 1
    Makefile:47: recipe for target 'ci_ct' failed
    make: *** [ci_ct] Error 1
    <F0><9F><9A><A8> Error: The command exited with status 1
```